src/components/budget/budget.service.ts

```diff
@@ -313,7 +313,7 @@ export class BudgetService {
 
     const records = await this.listRecords(
       {
-        sort: 'fiscalYear',
+        sort: 'createdAt',
```

Comment:
sorting on createdAt because fiscalYear is not a required field, and the new default sorter will not list records with null sort values

Reply:
Disagree. fiscalYear is the default. If they can't see the fiscalYear then the records are just unsorted that way, but they'd never know since they can't see the actual value.